### PR TITLE
Adding support for tables and columns list caching

### PR DESCRIPTION
**Pull request by fernandocarletti**
Hello,

Well, I did this little trick to save some queries on a production environment.

Now all tables and columns will be cached in the cache mechanism of user choice.

### CODE REVIEW

File: config/database.php [hunk 1]
```diff
@@ -24,9 +24,14 @@
 			'password'   => FALSE,
 			'persistent' => FALSE,
 		),
-		'table_prefix' => '',
-		'charset'      => 'utf8',
-		'caching'      => FALSE,
+        'table_prefix'     => '',
```
Comment (shadowhand) [inline config/database.php]: Whitespace looks broken.

File: config/database.php [hunk 2]
```diff
@@ -49,8 +54,13 @@
 		 *
 		 * string   identifier  set the escaping identifier
 		 */
-		'table_prefix' => '',
-		'charset'      => 'utf8',
-		'caching'      => FALSE,
+        'table_prefix'     => '',
```
Comment (shadowhand) [inline config/database.php]: Ditto.

**Comment by fernandocarletti** [comment]
Ooops... Fixed it!

**Comment by shadowhand** [comment]
This doesn't belong in the Database layer.

**Pull request closed without merging**